index.html

```diff
@@ -187,25 +190,28 @@ <h2>
 
       <section>
         <h2>
-Method-specific identifier
+Method-Specific Identifier
         </h2>
         <p>
 The method specific identifier is a fully qualified domain name that is
-secured by a TLS/SSL certificate with an optional path to the DID document.
-The formal rules describing valid domain name syntax are described in
-[[RFC1035]], [[RFC1123]], and [[RFC2181]].
+secured by a TLS/SSL certificate with an optional path to the DID document or
+the directory containing the DID document. The formal rules describing valid
+domain name syntax are described in [[RFC1035]], [[RFC1123]], and [[RFC2181]].
         </p>
         <p>
 The method specific identifier MUST match the common name used in the SSL/TLS
 certificate, and it MUST NOT include IP addresses. A port MAY be included and
 the colon MUST be percent encoded to prevent a conflict with paths. Directories
 and subdirectories MAY optionally be included, delimited by colons rather than
-slashes.
+slashes. Colons in directories and subdirectories MUST be percent encoded, if
+present, to avoid being translated into slashes. A filename MAY optionally be
+included as the final section of the colon delimited method specific identifier.
```

Comment:
```suggestion
slashes. A filename MAY optionally be included as the final section of the colon 
delimited method specific identifier. All path fragments (directories and 
filenames) MUST be percent encoded.
```
I think we should emphasize that all characters in path fragments are percent-encoded, not just colons.

Reply:
Colons are a special case. If a filename or path contains a literal `:`, that would be translated into `/` if not percent encoded.

If the colon was not our delimiter, it would not need percent encoding.

Reply:
@gribneau 
> If the colon was not our delimiter, it would not need percent encoding.

Ah, we still would, though, because directory and file names can contain characters that are not allowed by the DID spec. So it's not just the colon.

Reply:
The `:` that appears in the domain segment of the method specific identifier in the case that a port is specified must be percent-encoded to avoid being rewritten as `/`.

e.g.
```
did:web:example.com:1443
---> https://example.com/1443/

vs.

did:web:example.com%3A1443
---> https://example.com:1443/.well-known/did/
```